ui/appui-react/src/appui-react/icons/AssetClassificationHide.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Utilities
+ */
+
+import * as React from "react";
+
+/** @internal */
+export function AssetClassificationHide() {
```

Comment:
Sorry for missing this in my initial review, but I think we should prefix every SVG icon to stay consistent with `@itwin/itwinui-icons-react`
```suggestion
export function SvgAssetClassificationHide() {
```

Reply:
Should file names also include `Svg`?

Reply:
Yes, the component name should match the file name (even though it's not the case for iTwinUI icons).

Reply:
Updated this.